views/js/pciCreator/ims/mathEntryInteraction/runtime/mathEntryInteraction.js

```diff
@@ -443,26 +478,33 @@ define([
             /**
              * Gap mode only: fill the mathfield markup with the math expression before creating the MathQuill instance
              * @param {String} latex - the math expression with gaps
+             * @param {String} index
              */
-            setMathStaticContent: function setMathStaticContent(latex) {
+            setMathStaticContent: function setMathStaticContent(latex, index) {
+                const [inputIndex] = this.inputs.keys();
+                if (typeof index === 'undefined') {
+                    index = inputIndex;
+                }
```

Comment:
**suggestion:** As the same code is repeated in several places, it could have been a good idea to put it in a helper function
```suggestion
                function getIndex(inputs, index) {
                    if (typeof index === 'undefined') {
                        const [inputIndex] = inputs.keys();
                        return inputIndex;
                    }
                    return index;
```